### DISCUSSION TITLE
symmertric9 fill method with in a 2D simulation

### DISCUSSION THREAD

**Discussion by pharshaad** [idaholab/moose#18620] [Q&A Modules: Solid mechanics] [answered]
Hi all,

I was wondering if I could use the symmetric9 fill method with a 2D mesh.

Thanks,
Ari

**Comment by hugary1995** [comment, marked as answer]
Yes

**Reply by dschwen** [reply]
To eleborate, our tensors and vectors are _always_ 3D, no matter the mesh.

**Reply by pharshaad** [reply]
Got it, thanks, @hugary1995 and @dschwen.
